corehq/apps/userreports/tests/test_columns.py

```diff
@@ -175,9 +179,7 @@ def _new_case(self, properties):
             update=properties,
         ).as_xml()
         post_case_blocks([case_block], {'domain': self.domain})
-        case = CommCareCase.get(id)
-        self.addCleanup(case.delete)
-        return case
+        return CaseAccessors(self.domain).get_case(id)
```

Comment:
do we no longer need to `addCleanup` for this? I guess the table clears itself after each test class or something already?

Reply:
In this test I think that's probably true. For any test that's run on the sharded test runner we can't rely on the test transactions to clean up test data.